composer.json

```diff
@@ -9,14 +9,18 @@
         }
     ],
     "require": {
-        "php": "^7.3|^8.0"
+        "php": "^7.4|^8.0"
```

Comment:
Revert this change and rebase you PR to main - I've changed error with invalid return types few hours ago